ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/RecordValueTabulator.java

```diff
@@ -25,4 +26,5 @@
 public interface RecordValueTabulator<V> {
     Map<String, Object> toTabularValue(ConsumerRecord<?, V> record);
     Map<String, Object> getAttributes(ConsumerRecord<?, V> record);
+    Schema getSchema(ConsumerRecord<?, V> record);
```

Comment:
This method breaks contract, we can't expect all the implementations to provide Avro schema, what if we want to provide Json or Protobuf schema? I guest this method should be more generic, f.e. String getSchema(...);